app/controllers/benefits/utilities/EmployerProvidedServicesBenefitsAmountController.scala

```diff
@@ -76,11 +76,10 @@ class EmployerProvidedServicesBenefitsAmountController @Inject()(implicit val cc
                   utilitiesAndServices.map(_.copy(employerProvidedServices = Some(newAmount))))))
                 employmentSessionService.createOrUpdateSessionData(employmentId, updatedCyaModel, taxYear,
                   cya.isPriorSubmission, cya.hasPriorBenefits)(errorHandler.internalServerError()) {
-                  if (cya.isPriorSubmission) {
-                    Redirect(CheckYourBenefitsController.show(taxYear, employmentId))
-                  } else {
-                    Redirect(ProfessionalSubscriptionsBenefitsController.show(taxYear, employmentId))
-                  }
+
```

Comment:
There is a warning on line 46. If there is a new line between clock: Clock and ')' this will be fixed.